From f7e74a96da5ce89fa4b6d77f0407fe011fe8cd2f Mon Sep 17 00:00:00 2001
From: Jiayu Du <jiayu.riscv@isrc.iscas.ac.cn>
Date: Sat, 27 Dec 2025 17:23:54 +0800
Subject: [PATCH 00294/15333] riscv: dts: canaan: Correct the formatting issues
 about k230 dts

Correct the errors in the spacing format, and move the aliases from
the SoC's dtsi to the board's dts, since it's more standard.

Signed-off-by: Jiayu Du <jiayu.riscv@isrc.iscas.ac.cn>
Signed-off-by: Conor Dooley <conor.dooley@microchip.com>
---
 arch/riscv/boot/dts/canaan/k230-canmv.dts | 4 ++++
 arch/riscv/boot/dts/canaan/k230-evb.dts   | 4 ++++
 arch/riscv/boot/dts/canaan/k230.dtsi      | 6 +-----
 3 files changed, 9 insertions(+), 5 deletions(-)

diff --git a/arch/riscv/boot/dts/canaan/k230-canmv.dts b/arch/riscv/boot/dts/canaan/k230-canmv.dts
index 9565915cead6..593ca38e6824 100644
--- a/arch/riscv/boot/dts/canaan/k230-canmv.dts
+++ b/arch/riscv/boot/dts/canaan/k230-canmv.dts
@@ -9,6 +9,10 @@ / {
 	model = "Canaan CanMV-K230";
 	compatible = "canaan,canmv-k230", "canaan,kendryte-k230";
 
+	aliases {
+		serial0 = &uart0;
+	};
+
 	chosen {
 		stdout-path = "serial0:115200n8";
 	};
diff --git a/arch/riscv/boot/dts/canaan/k230-evb.dts b/arch/riscv/boot/dts/canaan/k230-evb.dts
index f898b8e62368..bfa53f2e2403 100644
--- a/arch/riscv/boot/dts/canaan/k230-evb.dts
+++ b/arch/riscv/boot/dts/canaan/k230-evb.dts
@@ -9,6 +9,10 @@ / {
 	model = "Kendryte K230 EVB";
 	compatible = "canaan,k230-usip-lp3-evb", "canaan,kendryte-k230";
 
+	aliases {
+		serial0 = &uart0;
+	};
+
 	chosen {
 		stdout-path = "serial0:115200n8";
 	};
diff --git a/arch/riscv/boot/dts/canaan/k230.dtsi b/arch/riscv/boot/dts/canaan/k230.dtsi
index 104e08ef5869..1aeb77160216 100644
--- a/arch/riscv/boot/dts/canaan/k230.dtsi
+++ b/arch/riscv/boot/dts/canaan/k230.dtsi
@@ -11,10 +11,6 @@ / {
 	#size-cells = <2>;
 	compatible = "canaan,kendryte-k230";
 
-	aliases {
-		serial0 = &uart0;
-	};
-
 	cpus {
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -75,7 +71,7 @@ soc {
 		ranges;
 
 		plic: interrupt-controller@f00000000 {
-			compatible = "canaan,k230-plic" ,"thead,c900-plic";
+			compatible = "canaan,k230-plic", "thead,c900-plic";
 			reg = <0xf 0x00000000 0x0 0x04000000>;
 			interrupts-extended = <&cpu0_intc 11>, <&cpu0_intc 9>;
 			interrupt-controller;
-- 
2.55.0

